src/test/java/de/retest/ui/image/ImageUtilsTest.java

```diff
@@ -0,0 +1,215 @@
+	}
+
+	@Test
+	public void marking_until_imageborder_should_increase_imagesize() throws IOException {
+		final BufferedImage image = ImageUtils.readImage( LOGIN_PNG );
+		final BufferedImage marked = ImageUtils.mark( image, new Rectangle( 0, 0, LOGIN_WIDTH, 57 ) );
+		// ImageUtils.exportImage(marked, new File(FileUtils.createTempFolder(),
+		// "login_marked_Panel.png"));
+		assertThat( marked.getWidth() ).isEqualTo( LOGIN_WIDTH + 2 * MARKING_WIDTH );
+		assertThat( marked.getHeight() ).isEqualTo( LOGIN_HEIGHT + MARKING_WIDTH );
+	}
+
+	@Test
+	public void marking_bigger_than_image_should_increase_imagesize() throws IOException {
+		final BufferedImage image = ImageUtils.readImage( LOGIN_PNG );
+		final BufferedImage marked = ImageUtils.mark( image, new Rectangle( 1, 0, 10, 10 ) );
+		// ImageUtils.exportImage(marked, new File(FileUtils.createTempFolder(),
+		// "login_marked_Panel.png"));
+		assertThat( marked.getWidth() ).isEqualTo( 256 + MARKING_WIDTH - 1 );
+	}
+
+	@Test
+	public void multiple_markings_should_not_increase_imagesize_further() throws IOException {
+		final BufferedImage image = ImageUtils.readImage( LOGIN_PNG );
+		final List<Rectangle> markings = new ArrayList<Rectangle>();
+		markings.add( new Rectangle( 0, 0, LOGIN_WIDTH, 10 ) );
+		markings.add( new Rectangle( 0, 10, LOGIN_WIDTH, 20 ) );
+		markings.add( new Rectangle( 0, 20, LOGIN_WIDTH, 30 ) );
+		final BufferedImage marked = ImageUtils.mark( image, markings );
+		assertThat( marked.getWidth() ).isEqualTo( LOGIN_WIDTH + 2 * MARKING_WIDTH );
+		assertThat( marked.getHeight() ).isEqualTo( LOGIN_HEIGHT + MARKING_WIDTH );
+	}
+
+	@Test
+	public void cut_image_should_have_bounds_length() throws IOException {
+		final BufferedImage image = ImageUtils.readImage( LOGIN_PNG );
+		final BufferedImage cut = ImageUtils.cutImage( image, new Rectangle( 10, 10, 20, 20 ) );
+		assertThat( cut.getWidth() ).isEqualTo( 20 );
+		assertThat( cut.getHeight() ).isEqualTo( 20 );
+	}
+
+	@Test
+	public void high_cutToMax_image_should_have_max_height_unchanged_width() throws IOException {
+		final BufferedImage image = new BufferedImage( 50, 10000, BufferedImage.TYPE_INT_ARGB );
+		final BufferedImage cut = ImageUtils.cutToMax( image );
+		assertThat( cut.getWidth() ).isEqualTo( 50 );
+		assertThat( cut.getHeight() ).isEqualTo( ImageUtils.MAX_SCREENSHOT_HEIGHT_DEFAULT );
+	}
+
+	@Test
+	public void smalles_possible_height_should_work() throws Exception {
+		final BufferedImage image = ImageUtils.readImage( LOGIN_PNG );
+		final BufferedImage cut = ImageUtils.cutImage( image, new Rectangle( 0, image.getHeight() - 1, 20, 20 ) );
+		assertThat( cut.getHeight() ).isEqualTo( 1 );
+	}
+
+	@Test
+	public void exact_height_should_return_null() throws Exception {
+		final BufferedImage image = ImageUtils.readImage( LOGIN_PNG );
+		final BufferedImage cut = ImageUtils.cutImage( image, new Rectangle( 0, image.getHeight(), 20, 20 ) );
+		Assertions.assertThat( cut ).isNull();
```

Comment:
I know it's just copy & paste, but would you mind making these imports (e.g. `Assertions` and `Mockito`) `static`? Some tests already have this.

Reply:
Können wir das jeweils einzeln, mit einzelnen PRs machen?
Wenn wir was nachvollziehen oder reverten wollen ist es dann einfacher. Außerdem hat das nicht zwingend etwas miteinander zu tun. Und wir wollen ja kleine Inkrements, kleine Commits und kleine Reviews...